chain/rosetta-rpc/src/adapters/transactions.rs

```diff
@@ -3,18 +3,21 @@ use std::string::ToString;
 
 use actix::Addr;
 
+use near_account_id::AccountId;
 use near_primitives::hash::CryptoHash;
 use near_primitives::views::SignedTransactionView;
 
+use crate::models::{AccountIdentifier, OperationMetadata};
+
 /// A mapping from NEAR transaction or receipt hash to list of receipts.
 /// and a mapping from transaction hashes to transactions.
 /// The latter map is needed to determine the amount of deposit in a single transaction when
 /// converting blocks to Rosetta transactions.
 pub(crate) struct ExecutionToReceipts {
     map: HashMap<CryptoHash, Vec<CryptoHash>>,
     transactions: HashMap<CryptoHash, SignedTransactionView>,
+    receipts: HashMap<CryptoHash, AccountId>,
```

Comment:
Could you document this field?  From documentation of the struct it’s no longer clear what this is.